Static Framework Sample/static-framework-sample-app/Showcases/ErrorAnalysis/ErrorAnalysisViewController.swift

```diff
@@ -0,0 +1,64 @@
+
+import ContentsquareErrorAnalysisModule
+import UIKit
```

Comment:
@Slorq Did you test if all the error analysis samples work as expected?

Reply:
You mean all the methods from `URLSession`?

Reply:
All the [IBAction]s in [ErrorAnalysisViewController.swift](https://github.com/ContentSquare/iOS-sample-app/pull/36/files/a5f408df38e4cbae20d86d010d0e6f3ca8184bcf#diff-9c37896219f25ae455d3c25de278aec65c885c24371ecd4add3833cb7942d0b7).

Reply:
Yes, I did test all the IBActions in all the sample apps